### PR TITLE
Add time out error code.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

**Review by Aiee — APPROVED** [review]

File: src/common/graph/Response.h [hunk 1]
```diff
@@ -48,6 +48,7 @@
     X(E_REBUILD_INDEX_FAILED            , -28)  \
     X(E_INVALID_PASSWORD                , -29)  \
     X(E_FAILED_GET_ABS_PATH             , -30)  \
+    X(E_RPC_TIMEOUT                     , -31)  \
```
Comment (laura-ding) [inline src/common/graph/Response.h:51]: add comment is the error code just for cpp client

Comment (Shylock-Hg) [reply src/common/graph/Response.h:51]: This could used by others too.

**Review by Aiee — APPROVED** [review]

**Comment by Aiee** [comment]
Is it necessary to re-generate thrift files for clients?

**Comment by Shylock-Hg** [comment]
> Is it necessary to re-generate thrift files for clients?

Don't force.

**Pull request closed without merging**